packages/synthetix-main/contracts/modules/core/VaultModule.sol

```diff
@@ -190,43 +192,56 @@ contract VaultModule is IVaultModule {
     }
 
     /**
-     * @dev See {IVaultModule-getVaultDebt}.
+     * @dev Returns the total debt for the given pool, and collateral type.
```

Comment:
same here, function is mutable becuase it triggers a debt distribution change in order to get the debt

Reply:
Implemented